config/default/views.view.legislation_listing.yml

```diff
@@ -0,0 +1,602 @@
+            link_to_entity: false
+          group_column: value
+          group_columns: {  }
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+          entity_type: node
+          entity_field: title
+          plugin_id: field
+      arguments:
+        title:
+          id: title
+          table: node_field_data
+          field: title
+          relationship: none
+          group_type: group
+          admin_label: ''
+          default_action: ignore
+          exception:
+            value: all
+            title_enable: true
+            title: All
+          title_enable: false
+          title: ''
+          default_argument_type: fixed
+          default_argument_options:
+            argument: a
+          default_argument_skip_url: false
+          summary_options:
+            base_path: ''
+            count: true
+            items_per_page: 25
+            override: false
+          summary:
+            sort_order: asc
+            number_of_records: 0
+            format: default_summary
+          specify_validation: true
+          validate:
+            type: none
+            fail: 'not found'
+          validate_options: {  }
+          glossary: true
+          limit: 1
+          case: upper
+          path_case: lower
+          transform_dash: false
+          break_phrase: false
+          plugin_id: string
+          entity_type: node
+          entity_field: title
+      relationships: {  }
+      style:
+        type: html_list
+        options:
+          grouping:
+            -
+              field: title_1
+              rendered: true
+              rendered_strip: false
+          row_class: ''
+          default_row_class: true
+          type: ul
+          wrapper_class: item-list
+          class: ''
+      row:
+        type: fields
+        options:
+          inline: {  }
+          separator: ''
+          hide_empty: false
+          default_field_elements: true
+      header:
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: false
+          tokenize: false
+          content:
+            value: "<a href=\"../legislation\">Namibia Consolidated Acts Index Home Page</a>\r\n\r\nThis collection of consolidated legislation was initially prepared by the Namibian Government. <a href=\"https://laws.parliament.na/about-us/\"> Find out more here. </a> </br>\r\n\r\nThe pre- and post-independence annotated laws as well as the post-independence regulations were prepared by the Legal Assistance Centre in Namibia. </br>\r\n\r\nAmendments post November 2017 are applied on a continuous basis by the <a href=\"https://laws.africa/\"> Laws.Africa team and contributors. </a> </br>"
```

Comment:
This site is for all LIIs, so we can't have this Namibia-specific header here. Let's just remove it completely for now.

Reply:
Although I don't see it showing up in my view, so 🤷

Reply:
Cool will remove that.